react/week3/todo-list/src/App.js

```diff
@@ -0,0 +1,82 @@
+import { useState, useEffect } from "react";
+import "./App.css";
+import TodoList from "./TodoList";
+import Fetching from "./Fetching";
+import { getNextId } from "./Helper";
+import Timer from "./Timer";
```

Comment:
It's really good to see that you are really starting to use different components, breaking helper functions into helper files etc. Well done! Not much that I would do a whole lot different, just minor things here and there. Great homework solution.